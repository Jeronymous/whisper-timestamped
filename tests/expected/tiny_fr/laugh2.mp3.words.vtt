WEBVTT

00:00.160 --> 00:00.420
Merci

00:00.420 --> 00:00.820
beaucoup.

