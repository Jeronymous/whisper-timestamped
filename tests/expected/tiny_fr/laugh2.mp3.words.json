{
  "text": " Merci beaucoup.",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.16,
      "end": 0.82,
      "text": " Merci beaucoup.",
      "tokens": [
        50364,
        19856,
        8796,
        13,
        50414
      ],
      "temperature": 0.0,
      "avg_logprob": -1.3000399271647136,
      "compression_ratio": 0.6521739130434783,
      "no_speech_prob": 0.43056344985961914,
      "words": [
        {
          "text": "Merci",
          "start": 0.16,
          "end": 0.42
        },
        {
          "text": "beaucoup.",
          "start": 0.42,
          "end": 0.82
        }
      ]
    }
  ],
  "language": "fr"
}